Library/Formula/syncthing-inotify.rb

```diff
@@ -15,22 +15,11 @@ class SyncthingInotify < Formula
 
   depends_on "go" => :build
 
-  go_resource "github.com/cenkalti/backoff" do
-    url "https://github.com/cenkalti/backoff.git",
-        :revision => "6c45d6bc1e78d94431dff8fc28a99f20bafa355a" # not sure !
-  end
-
-  go_resource "github.com/zillode/notify" do
-    url "https://github.com/Zillode/notify.git",
-      :revision => "f06b1e3b795091f2e1414067b08e5f07332cdb05"   # not sure !
-  end
-
   def install
     ENV["GOPATH"] = buildpath
-    ENV.append_path "PATH", buildpath
     bin_name = "syncthing-inotify"
-    Language::Go.stage_deps resources, buildpath/"src"
-    system "go", "build", "-o", bin_name
+    system "go", "get", "-d"
```

Comment:
Please keep using our system of `go_resources` here. We don't use `go get` directly in formulae.